recipe/meta.yaml

```
@@ -45,39 +55,42 @@ outputs:
         - make  # [unix]
       host:
         - zlib
+        - gmp
```

```suggestion
        - zlib {{ zlib }}
        - gmp  {{ gmp }}
```

Or `zlib {{ libzlib }}`?? Here and below in other outputs